celements-model/src/main/java/com/celements/model/access/DefaultXWikiDocumentCreator.java

```diff
@@ -47,8 +52,12 @@ public XWikiDocument createWithoutDefaults(DocumentReference docRef) {
 
   @Override
   public XWikiDocument create(DocumentReference docRef, String lang) {
+    String defaultLang = getDefaultLangForCreatingDoc(docRef);
+    if (defaultLang.equals(lang)) {
```

Comment:
this misses the "normalizeLang" call. After which it could again match the defaultLang